### PR TITLE
fix(PublicShareAuthSidebar): fix and refactor styles for video verification sidebar

### PR DESCRIPTION
**Pull request by Antreesy**
### ☑️ Resolves

* Fix broken styles
* apply `box-sizing: border-box`
* adjust spacings
* fix colors and text appearance

### 🖼️ Screenshots / Screencasts

🏚️ Before | 🏡 After
-- | --
image | Screenshot from 2024-07-30 13-56-40
Auth | --
image | image

### 🏁 Checklist

- [ ] 🌏 Tested with Chrome, Firefox and Safari or should not be risky to browser differences
- [ ] 🖥️ Tested with Desktop client or should not be risky for it 
- [ ] 🖌️ Design was reviewed, approved or inspired by the design team
- [ ] ⛑️ Tests are included or not possible
- [ ] 📗 User documentation in https://github.com/nextcloud/documentation/tree/master/user_manual/talk has been updated or is not required

### CODE REVIEW

File: src/PublicShareAuthSidebar.vue [hunk 1]
```diff
@@ -204,16 +204,24 @@ export default {
...
-/* Properties based on the app-sidebar */
+/* Styles based on the NcAppSidebar */
 #talk-sidebar {
 	position: relative;
 	flex-shrink: 0;
-	width: 27vw;
-	min-width: 300px;
-	max-width: 500px;
+	width: clamp(300px, 27vw, 500px);
+	height: 100%;
```
Comment (DorraJaouad) [inline src/PublicShareAuthSidebar.vue:224]: No parent has a determined height. You can set` height: 100%` in `body.talk-sidebar-enabled` or use `vh` to not be dependent.

Comment (Antreesy) [reply src/PublicShareAuthSidebar.vue:224]: Adjusted in external css, please give it a try again.
I've tested it in FF in Chrome

File: src/components/TopBar/CallButton.vue [hunk 2]
```diff
@@ -178,7 +178,15 @@ export default {
...
-		 * Whether the to use text on button at mobile view
+		 * Whether to use text on button (e.g. at sidebar)
+		 */
+		hideText: {
+			type: Boolean,
+			default: false,
+		},
+
+		/**
+		 * Whether to use text on button at mobile view
```
Comment (DorraJaouad) [inline src/components/TopBar/CallButton.vue:189]: Let's also use it for chat sidebar in Files?

**Review by DorraJaouad — APPROVED** [review]
Looks good 🌟

File: src/components/TopBar/CallButton.vue (hunk 2 above)
Comment (Antreesy) [reply src/components/TopBar/CallButton.vue:189]: It's already there, but for the TopBar. Button in the chat has enough space to show the text

Comment (DorraJaouad) [reply src/components/TopBar/CallButton.vue:189]: oops, pointed to the wrong line :
`<CallButton v-if="!isInCall"`

so call button can be removed 
image

**Pull request merged**